ApplicationPattern.yaml

```diff
@@ -600,6 +636,7 @@ paths:
                         - application-release-number
                         - embedding-operation
                         - client-update-operation
+                        - application-protocol
```

Comment:
line 620 : callback uri definition for PromptForRegisteringCausesRegistrationRequest has protocol https which needs to be updated to reference registry-office-protocol 
{$request.body#registry-office-protocol}://...